src/test/java/marquez/api/JobTest.java

```diff
@@ -33,42 +33,46 @@ public void testGuidSet() {
 
   @Test
   public void testJobEquality() {
-    Job j2 = new Job(JOB_UUID, JOB_NAME, OWNER_NAME, NOMINAL_TIME, CATEGORY, DESCRIPTION);
+    Job j2 = new Job(JOB_UUID, JOB_NAME, OWNER_NAME, NOMINAL_TIME, CATEGORY, DESCRIPTION, null);
     AssertionsForClassTypes.assertThat(JOB.equals(JOB));
     AssertionsForClassTypes.assertThat(JOB.equals(j2));
     AssertionsForClassTypes.assertThat(j2.equals(JOB));
   }
 
   @Test
   public void testHashCodeEquality() {
-    Job j2 = new Job(JOB_UUID, JOB_NAME, OWNER_NAME, NOMINAL_TIME, CATEGORY, DESCRIPTION);
+    Job j2 = new Job(JOB_UUID, JOB_NAME, OWNER_NAME, NOMINAL_TIME, CATEGORY, DESCRIPTION, null);
     assertEquals(JOB.hashCode(), j2.hashCode());
   }
 
   @Test
   public void testJobInequalityOnUUID() {
-    Job j2 = new Job(UUID.randomUUID(), JOB_NAME, OWNER_NAME, NOMINAL_TIME, CATEGORY, DESCRIPTION);
+    Job j2 =
+        new Job(UUID.randomUUID(), JOB_NAME, OWNER_NAME, NOMINAL_TIME, CATEGORY, DESCRIPTION, null);
```

Comment:
Let's adjust the tests to remove references to category, since that's going to be removed.

Reply:
I didn't touch these API model tests too much, mostly tried to keep them as-is and work around them

Reply:
Reverted these changes back to their original state (along with the API models themselves). Since you're working with the job resource layer, you'll be modifying these API models anyway, so don't want to collide.

Reply:
Ok no worries - I can handle those.